yearclass/src/main/java/com/facebook/device/yearclass/YearClass.java

```diff
@@ -60,6 +71,7 @@ private static void conditionallyAdd(ArrayList<Integer> list, int value) {
    * (specifically app startup time, scrolling perf, animations) are more uniform within
    * the buckets than with the 2014 calculations.
    */
+  @Class
   private static int categorizeByYear2016Method(Context c) {
```

Comment:
I prefer private static @Class int categorizeByYear2016Method, since the @Class is referencing the int.

For example the other annotation is for the whole method, so @Class just doesn't fit there.